tomopy/prep/alignment.py

```diff
@@ -443,3 +455,240 @@ def shift_images(prj, sx, sy):
     prj *= scl
 
     return prj
+
+
+def find_slits_corners_aps_1id(img, 
+                               method='quadrant+', 
+                               medfilt2_kernel_size=3,
+                               medfilt_kernel_size=23,
+                               ):
+    """
+    Automatically locate the slit box location by its four corners.
+
+    Parameters
+    ----------
+    img                   :  np.2darray
+        input 2D images
+    method                :  str,  ['simple', 'quadrant', 'quadrant+'], optional
+        method for auto detecting slit corners
+    medfilt2_kernel_size  :  int, optional 
+        2D median filter kernel size for noise reduction
+    medfilt_kernel_size   :  int, optional 
+        1D median filter kernel size for noise reduction

```

Comment:
What is the function of 1D and 2D median filters?

Reply:
2D median filter:  remove noise points in the image due to defect defects

1D median filter:  reduce the local fluctuation of the 1D gradient profile to avoid miss-identification of the slit location